skbio/stats/composition.py

```diff
@@ -139,6 +139,8 @@ def closure(mat):
         raise ValueError("Cannot have negative proportions")
     if mat.ndim > 2:
         raise ValueError("Input matrix can only have two dimensions or less")
+    if np.all(mat == 0, axis=1).sum() > 0:
+        raise ValueError("Input matrix cannot have rows with all zeros")
```

Comment:
@mortonjt, can you add this and the rest of the exceptions to a `Raises` section in the docstring.